logdnaHandlerTest.py

```diff
@@ -0,0 +1,151 @@
+from http.server import BaseHTTPRequestHandler,HTTPServer
+import logging
+import threading
+from logdna import LogDNAHandler
+import unittest
+import json
+import concurrent.futures
+import time
```

Comment:
PEP8 recommends grouping and ordering imports by standard / 3rd-party / local. I recommend we do that. Also alphabetize within each group.